src/lib/drishti/core/ImageView.h

```diff
@@ -0,0 +1,46 @@
+/*!
+  @file   drishti/core/ImageView.h
+  @author David Hirvonen
+  @brief Container class for multiple views of the same image (RAM, GPU)
+
+  \copyright Copyright 2017 Elucideye, Inc. All rights reserved.
+  \license{This project is released under the 3 Clause BSD License.}
+
+*/
+
+#ifndef __drishti_core_ImageView_h__
+#define __drishti_core_ImageView_h__
+
+#include <opencv2/core.hpp> // for cv::Mat4b
+
+DRISHTI_CORE_NAMESPACE_BEGIN
+
+struct Texture
+{
+    Texture() = default;
+    Texture(const cv::Size &size, GLuint texId)
```

Comment:
Android error (+Qt): `GLuint has not been declared`

Reply:
Added #561